addons/project_todo/views/project_task_views.xml

```diff
@@ -7,63 +7,44 @@
                     </t>
-                    <t t-name="kanban-box">
+                    <t t-name="kanban-card">
                         <t t-set="todoHasAssignees" t-value="record.user_ids.raw_value.length &gt; 1"/>
-                        <div t-attf-class="#{kanban_color(record.color.raw_value)} oe_kanban_global_click_edit oe_semantic_html_override oe_kanban_card">
-                            <div class="oe_kanban_content">
-                                <div class="o_kanban_record_top">
-                                    <div class="o_kanban_record_headings">
-                                        <div class="d-flex justify-content-start">
-                                            <div t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                                <strong class="o_kanban_record_title align-middle">
-                                                    <field name="name" widget="name_with_subtask_count"/>
-                                                </strong>
-                                            </div>
-                                        </div>
-                                    </div>
-                                </div>
-                                <div class="o_kanban_record_body o_todo_kanban_card_body" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                    <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
-                                    <div t-if="record.displayed_image_id.value">
-                                        <field name="displayed_image_id" widget="attachment_image"/>
-                                    </div>
-                                </div>
-                                <div class="o_kanban_record_bottom mt-2">
-                                    <div class="oe_kanban_bottom_left" t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
-                                        <field name="priority" widget="priority"/>
-                                        <field name="activity_ids" widget="kanban_activity" style="padding-top: 1.5px;"/>
-                                    </div>
-                                    <div class="oe_kanban_bottom_right">
-                                        <div t-att-class="{'opacity-50': ['1_done', '1_canceled'].includes(record.state.raw_value), 'o_todo_hide_avatar': !todoHasAssignees}">
-                                            <field name="user_ids" widget="many2many_avatar_user" readonly="True"/>
-                                        </div>
-                                        <field name="state" widget="todo_done_checkmark"/>
-                                    </div>
+                        <div t-att-class="['1_done', '1_canceled'].includes(record.state.raw_value) ? 'opacity-50' : ''">
```

Comment:
t-att-class can be simplified
t-att-class="{'opacity-50': ['1_done', '1_canceled'].includes(record.state.raw_value) }"

Reply:
same below